packages/shared/src/components/Markdown.tsx

```diff
@@ -35,63 +55,36 @@ export default function Markdown({
   );
 
   useEffect(() => {
-    if (!content || !ref?.current) {
-      return;
-    }
-
-    const elements = Array.from(ref.current.getElementsByTagName('a')).filter(
-      (element) => !!element.getAttribute('data-mention-id'),
-    );
-
-    if (elements.length === 0) {
+    if (data || !userId) {
       return;
     }
+    enableFetchInfo();
```

Comment:
We might not even need fetchInfo anymore since we just show devcard?

Reply:
We don't need the `USER_TOOLTIP_CONTENT_QUERY` anymore? I can remove this part if needed 👀

Reply:
Yeah I think we dont need it anymore since the data for devcard is now completely in a different format and it is fetched from the card itself now.

Reply:
Nice lets kill it 🔪